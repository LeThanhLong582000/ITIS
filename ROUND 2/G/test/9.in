20
100000111010000010000011000101111001011111101101010110110011
0011001110010
0110111111110111010011000110110101001111101010101010100
11111001100100000010100011100010110
1101010000110001000100001110000101101011110101100010011001011010001111101101
1010001011010010101010111101010101
110011010101
011111101101000101110011111111100101010101011011010000
1110110111111101110010100111000101110010111100
0011011101101101101000101101000101010111110001010110110010100000110110101101001010100001011001101
01101110100000
010011001011111011101010100001
1110100101001001001110010010100100101010011100001100001010100100110111010100111110010110111110011000
1101010000000001010101110100100111111111000110101101100101010111010001111001110001101110101111011
000111000101010111110001000110011011011110100101010010110011011110011101110
111101001001000101101110101001001111000111100011010100010001100111010100111110110010100111
0100111100110011000000100111111
111101110011100010001101
111011110001111110100110
110011111000111011001100100000010010010010011
