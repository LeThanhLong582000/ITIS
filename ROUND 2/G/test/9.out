19
5
20
11
30
14
5
18
15
43
3
11
41
36
31
41
10
7
7
15
